### PR TITLE
Attempt another round of dom update file

### PR DESCRIPTION
**Pull request by anewb87**
#### What's this PR do?
- successfully switched dom fns over to domUpdates.js
- some css refactoring for the table, filter bar, and headers

#### Where should the reviewer start?
domUpdates.js and scripts.js

#### How should this be manually tested?
open up the site - it should still be working properly after all of the file changes

#### Any background context you want to provide?
- the only thing left is refactoring to check for correct syntax and error handling button clicks 

#### Questions:

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -0,0 +1,452 @@
+// IMPORTS
```
Comment (anewb87) [inline src/domUpdates.js:1]: This was the definition of a doozy! After checking in with Hannah, we think we're at least on the right track here. And hey- project still functions!

**Review by anewb87 — COMMENTED** [review]
Seven hours later...

**Review by kammon10 — APPROVED** [review]

File: src/styles/index.scss [hunk 2]
```diff
@@ -116,20 +123,17 @@ h3 {
...
 
-.button-message {
-  font-size: 12px;
-}
 
 /* ~~~~~~~~~~ INGREDIENT TABLE ~~~~~~~~~~~~ */
 
 table {
-  width: 85%;
+  // width: 85%;
```
Comment (anewb87) [inline src/styles/index.scss:130]: Will remove. This helped us get our table more to the center of the page.

File: src/styles/index.scss [hunk 3]
```diff
@@ -151,9 +155,29 @@ td {
 
 }
 
+.button-message {
```
Comment (anewb87) [inline src/styles/index.scss:158]: These widths are keeping the table 'static' so they no longer move and cause a user annoyance when they add or subtract ingredients.

**Review by aliroemhildt — APPROVED** [review]
what a journey, but we finally got it. WOOHOO!

File: src/styles/index.scss [hunk 4]
```diff
@@ -191,6 +215,12 @@ nav {
 
 /* ~~~~~~~~~~ FILTER SECTION ~~~~~~~~~~~~ */
 
+.filter-title {
```
Comment (anewb87) [inline src/styles/index.scss:218]: This makes more sense with the design than how we had it before.

**Comment by cagallo** [comment]
- [x] Got it! Great organization, makes sense that we would need to query selectors, event listeners and helper functions outside of the dom object.

**Pull request merged**